### PR TITLE
Call deadlines

### PR DESCRIPTION
**Pull request by HelloGrayson**
Instead of hanging forever when a test client hangs, give them 5 seconds to respond.

cc @yarpc/yarpc @abhinav @prashantv

### CODE REVIEW

File: plan/config.go [hunk 1]
```diff
@@ -2,19 +2,26 @@ package plan
...
-	const axisKeyPrefix = "CROSSDOCK_AXIS_"
-
-	clients := strings.Split(os.Getenv(clientsKey), ",")
-	clients = trimCollection(clients)
+const callDeadlineDefault = 5
 
+// ReadConfigFromEnviron creates a Config by looking for CROSSDOCK_ environment vars
+func ReadConfigFromEnviron() *Config {
+	const (
+		callDeadlineKey = "CROSSDOCK_CALL_DEADLINE"
```
Comment (kriskowal) [inline plan/config.go]: Might be less confusing to call it TTL, since it’s a duration relative to start time. Deadline implies an absolute time.

**Comment by kriskowal** [comment]
:ok:

**Comment by jc-fireball** [comment]
it is one client is unreachable then all fail, or let the crossdock test continue with it.

**Comment by HelloGrayson** [comment]
@junchaowu just that one call will timeout, and the test will continue.

File: plan/config.go (hunk 1 above)
Comment (HelloGrayson) [inline plan/config.go]: Renaming to `CROSSDOCK_CALL_TIMEOUT`.

File: execute/run.go [hunk 2]
```diff
@@ -4,14 +4,17 @@ import (
...
 	"fmt"
 	"io/ioutil"
-	"net/http"
 	"net/url"
+	"time"
+
+	"golang.org/x/net/context"
+	"golang.org/x/net/context/ctxhttp"
 
 	"github.com/yarpc/crossdock/plan"
```
Comment (abhinav) [inline execute/run.go]: Nit: Fix import group ordering. It should be:

   std library

```
local packages

third party packages
```

**Comment by abhinav** [comment]
:+1:

**Pull request merged**